Solutions/Armorblox/Data Connectors/ArmorbloxAzureSentinelConnector/__init__.py

```diff
@@ -59,43 +61,29 @@ def generate_date():
         state.post(current_time.strftime(ARMORBLOX_INCIDENT_API_TIME_FORMAT))
         return past_time, current_time.strftime(ARMORBLOX_INCIDENT_API_TIME_FORMAT)
 
-    def _process_incidents(self, url, headers, params):
-        response = requests.get(url, headers=headers, params=params)
-        if response.status_code == 200:
-            response_json = response.json()
-            next_page_token = response_json.get("next_page_token", None)
-            self.incidents_list.extend(response_json.get("incidents", []))
-            if next_page_token is not None:
-                params["page_token"] = next_page_token
-                self._process_incidents(url, headers, params)
+
+    def _process_incidents(self, params):
+        response_json,_ = self.incidents.list_resource('incidents', params=params)
```

Comment:
Can we use `self.incidents.list` directly instead of calling `list_resource`?

Reply:
We need to query with the parameters and `incidents.list()` method does not allow that. So, we need to call `list_resource`